PubSub/src/Connection/Rest.php

```diff
@@ -298,4 +298,64 @@ public function detachSubscription(array $args)
     {
         return $this->send('subscriptions', 'detach', $args);
     }
+
+    /**
+     * @param array $args
+     */
+    public function listSchemas(array $args)
+    {
+        throw new \RuntimeException(
```

Comment:
Are these all unimplemented because we don't use the rest transport anymore?

Reply:
We still use it. They're unimplemented because the public json schema isn't updated to include the new endpoints and objects.

Reply:
Is that something that is supposed to be auto-generated or something else we need to do manually?

Reply:
To be honest, I'm not sure! It is often absent for pre-GA features. I don't know whether that is something that happens automatically or if it is triggered by a human.

Reply:
Is [this](https://pubsub.googleapis.com/$discovery/rest?version=v1) what you're referring to? If so, it has the schema resources now